src/constants.ts

```diff
@@ -1 +1,36 @@
 export const BLOCKCHAIN_PROTOCOL_VERSION = '1.0.0';
+
+export const EVENT_HANDLER_ERROR_CODE = {
```

Comment:
I think having two places for the full error code list (in the blockchain code and here) would increase the maintenance cost. what about just having here only those needed, i.e., FAILED_TO_REGISTER_FILTER?

Reply:
Oh, it would be nice. i will fix it

Reply:
Fixed, thanks!